libraries/chain/webassembly/privileged.cpp

```diff
@@ -156,7 +156,7 @@ namespace eosio { namespace chain { namespace webassembly {
    struct abi_finalizer_authority {
       std::string              description;
       uint64_t                 fweight = 0; // weight that this finalizer's vote has for meeting fthreshold
-      std::array<uint8_t, 144> public_key_g1_jacobian;
+      std::array<uint8_t, 96>  public_key_g1_affine_le;
```

Comment:
Shouldn't this also include the proof of possession?

Reply:
Proof of possession will be done in contract. It will not be enforced in consensus.